plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -393,7 +393,7 @@ function BookInfoManager:extractBookInfo(filepath, cover_specs)
         -- so it does not reset our temporary cache dir when we first open
         -- a crengine book for extraction.
         require("document/credocument"):engineInit()
-        local cre = require "libs/libkoreader-cre"
+        --local cre = require "libs/libkoreader-cre"
```

Comment:
If you let it commented out, please add a comment:
`-- already loaded as a global by previous :engineIniti()`
if that's really what is happening and why it doesn't crash :)

Why/how did you get to change this ?

Reply:
`engineInit()` loads cre as a global.